.github/workflows/main.yml

```
@@ -22,10 +24,20 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: docker/build-push-action@v1.1.0
+      - name: Set up Docker Buildx
+        id: setup-buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          platforms: |
+            linux/amd64
+            linux/arm64
+
```

l'authentifcation n'est plus disponible en `docker/build-push-action@v4`

```suggestion

      - name: Login to Docker Hub
        uses: docker/login-action@v2
        with:
            username: ${{ secrets.DOCKER_USERNAME }}
            password: ${{ secrets.DOCKER_PASSWORD }}

```